### ISSUE TITLE
Not able to reproduce the result reported in the paper

### ISSUE THREAD

**Issue by avijit9** [vvanirudh/social-lstm-pytorch#2]
I am running the code exactly same as this repo. But I am getting larger error metrics. Can you please tell me the exact parameters you used that you used while reporting in the paper.

**Comment by vvanirudh** [comment]
Are you using the model with the least validation error among all epochs?

**Comment by avijit9** [comment]
Yes. I am using the model which has least validation error out of 150 epochs. Can you check the pixel to meter conversion code I posted here? I suspect that I am doing something wrong here. https://github.com/vvanirudh/srnn-pytorch/issues/1
